Manuscript/Text/02D_IL2dose.md

```diff
@@ -1,9 +1,9 @@
 
-While the model accurately predicted experimentally-measured responses overall, and specifically the sensitivities of the dose-response profiles, we noticed some discrepancy specifically at high ligand concentrations and longer times in specific cell populations ([@Fig:expData]; [@Fig:supp5]). For example, while CD8+ cells almost exactly match model predictions at 1 hr, by 4 hrs we experimentally observed a biphasic response with respect to IL-2 concentration, and a plateau with IL-15 that decreased over time. This decrease in signaling was most pronounced with the CD8+ cells, but could be observed to lesser extents in some other cell populations such as NK cells ([@Fig:supp5]). We hypothesize two possible explanations for this discrepancy: First, CD8+ populations are known to proteolytically shed IL-2Rα in an activity-responsive manner [@Junghans1587]. Second, our model only uses a very simple sigmoidal relationship between active receptor and pSTAT5 signal. Other components of the JAK-STAT pathway surely influence its dynamic response [@Kuwabara20164334]. Our model also disagreed with experimental results for the pSTAT5 response of NK and CD8+ cell types at intermediate IL-2 concentrations, resulting in innacuracies in some EC~50~ predictions ([@Fig:expData]B, D-E). However, overall, the model presented here remains useful for exploring the determinants of cell type-specific response, which originate at the receptor expression profile on the cell surface.
+While the model accurately predicted experimentally-measured responses overall, and specifically the sensitivities of the dose-response profiles, we noticed some discrepancy specifically at high ligand concentrations and longer times in specific cell populations ([@Fig:expData]; [@Fig:supp5]). For example, while CD8+ cells almost exactly match model predictions at 1 hr, by 4 hrs we experimentally observed a biphasic response with respect to IL-2 concentration, and a plateau with IL-15 that decreased over time. This decrease in signaling was most pronounced with the CD8+ cells, but could be observed to lesser extents in some other cell populations such as NK cells ([@Fig:supp5]). We hypothesize two possible explanations for this discrepancy: First, CD8+ populations are known to proteolytically shed IL-2Rα in an activity-responsive manner [@Junghans1587]. Second, our model only uses a very simple scalar relationship between active receptor and pSTAT5 signal. Other components of the JAK-STAT pathway surely influence its dynamic response [@Kuwabara20164334]. Our model also disagreed with experimental results for the pSTAT5 response of NK and CD8+ cell types at intermediate IL-2 concentrations, resulting in innacuracies in some EC~50~ predictions ([@Fig:expData]B, D-E). However, overall, the model presented here remains useful for exploring the determinants of cell type-specific response, which originate at the receptor expression profile on the cell surface.
```

Comment:
A scalar relationship is implied, so I think you can simplify these two sentences and simply say we don't model the Jak-STAT5 pathway.